tests/test_dataset.py

```diff
@@ -382,7 +382,6 @@ def test_annotate_async(dataset: Dataset):
         "status": "Completed",
         "message": {
```

Comment:
This is a terrible testing pattern that we're still unwinding - we shouldn't be checking the message at all. We can test that in `scaleapi` unit tests if we want. 

Can change the test to use `assert_partial_equality` and ideally just check the `status` message or the subset that  indicates succes.